sql/hive/src/test/scala/org/apache/spark/sql/hive/MetastoreDataSourcesSuite.scala

```diff
@@ -622,7 +622,7 @@ class MetastoreDataSourcesSuite extends QueryTest with SQLTestUtils with TestHiv
         .mode(SaveMode.Append)
         .saveAsTable("arrayInParquet")
 
-      sessionState.refreshTable("arrayInParquet")
+      sparkSession.catalog.refreshTable("arrayInParquet")
```

Comment:
Actually, invalidateTable and refreshTable do have different meanings. The current implementation of `HiveMetastoreCatalog.refreshTable` is `HiveMetastoreCatalog.invalidateTable` (and then we retrieve the new metadata lazily). But, it does not mean that `refreshTable` and `invalidateTable` have the same semantic. If we should remove any of `invalidateTable` or `refreshTable` should be discussed in a different thread.